build.sh

```diff
@@ -5,3 +5,4 @@ docker build --build-arg http_proxy=$http_proxy --build-arg https_proxy=$https_p
  docker cp faas-cli:/root/faas-cli . && \
  docker rm -f faas-cli
 
+go test ./...
```

Comment:
What does this do? `./...`

Reply:
Runs all unit tests in the entire project, directory by directory. If it encounters a directory without tests, it just does nothing. It the dir has tests, it runs them.